io.gitlab.librewolf-community.json

```
@@ -0,0 +1,75 @@
+{
+    "app-id": "io.gitlab.librewolf-community",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "40",
+    "sdk": "org.gnome.Sdk",
```

```suggestion
    "runtime": "org.freedesktop.Platform",
    "runtime-version": "20.08",
    "sdk": "org.freedesktop.Sdk",
    "add-extensions": {
        "org.freedesktop.Platform.ffmpeg-full": {
            "directory": "lib/ffmpeg",
            "version": "20.08",
            "add-ld-path": "."
        }
    },
```

Got it from https://hg.mozilla.org/mozilla-central/file/tip/taskcluster/docker/firefox-flatpak/runme.sh#l97.

---

Do we need to include ffmpeg / what would it be useful for?
(Not arguing against it – I just hadn't missed it yet ;)

---

I genuinely don't know. I saw it on the Firefox manifest so I figured I should just add it. Do you think I should remove it?

---

I'd say: keep it, I'll go out on a limb here and assume Mozilla would've investigated that enough in the past to now save us some work. So unless someone with more insight in Flatpak-intricacies suggests it might be a bad idea, I'd keep it :)